build.gradle.kts

```diff
@@ -220,9 +262,30 @@ val makeTreeSitterPklLib by
 tasks.processResources {
   dependsOn(makeTreeSitterLib)
   dependsOn(makeTreeSitterPklLib)
+  // tree-sitter's CLI always generates debug symbols when on version 0.22.
+  // we can remove this when tree-sitter-pkl upgrades the tree-sitter-cli dependency to 0.23 or
+  // newer.
+  exclude("**/*.dSYM/**")
+  filesMatching("org/pkl/lsp/Release.properties") {
+    filter<ReplaceTokens>(
+      "tokens" to
+        mapOf(
+          "version" to buildInfo.pklLspVersion,
+          "treeSitterVersion" to libs.versions.treeSitterRepo.get(),
+          "treeSitterPklVersion" to libs.versions.treeSitterPklRepo.get(),
```

Comment:
Provide versions of these libraries to runtime, because this affects where these libraries get loaded from. For example, on macOS, the tree-sitter lib lives at `~/.pkl/editor-support/native-libs/tree-sitter/v0.23.0/libtree-sitter.dylib`.

This ensures that the version of the native lib matches what the LSP expects.